sdk/create-dapp/templates/react-client-dapp/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@mysten/template-react-client-app",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@mysten/dapp-kit": "workspace:*",
```

Comment:
Does this workspace directive get converted on the published package?

Reply:
these get overwritten when it writes the files from the template